substrate/frame/core-fellowship/src/lib.rs

```diff
@@ -100,29 +100,46 @@ pub enum Wish {
 pub type Evidence<T, I> = BoundedVec<u8, <T as Config<I>>::EvidenceSize>;
 
 /// The status of the pallet instance.
-#[derive(Encode, Decode, Eq, PartialEq, Clone, TypeInfo, MaxEncodedLen, RuntimeDebug)]
-pub struct ParamsType<Balance, BlockNumber, const RANKS: usize> {
+#[derive(
+	Encode,
+	Decode,
+	CloneNoBound,
+	EqNoBound,
+	PartialEqNoBound,
+	RuntimeDebugNoBound,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[scale_info(skip_type_params(Ranks))]
+pub struct ParamsType<
+	Balance: Clone + Eq + PartialEq + Debug,
+	BlockNumber: Clone + Eq + PartialEq + Debug,
+	Ranks: Get<u32>,
+> {
 	/// The amounts to be paid when a member of a given rank (-1) is active.
-	active_salary: [Balance; RANKS],
+	active_salary: BoundedVec<Balance, Ranks>,
```

Comment:
This still changes the encoding and requires some migration

Reply:
Currently figuring out try-runtime cli, migration underway